wsmaster/che-core-api-workspace/src/test/resources/.che.json

```diff
@@ -0,0 +1 @@
+Hello
```

Comment:
IMHO It's strange resource file. I think it makes sense to rename it to make it clearer what is its purpose and where it's used.

Reply:
fixed